hydrolib/core/io/polyfile/model.py

```diff
@@ -0,0 +1,28 @@
+from hydrolib.core.io.polyfile.components import PolyObject
+from hydrolib.core.io.polyfile.serializer import write_polyfile
+from hydrolib.core.io.polyfile.parser import read_polyfile
+from hydrolib.core.basemodel import FileModel
+from typing import Callable, Sequence
+
+
+class PolyFile(FileModel):
+    """Poly-file (.pol/.pli/.pliz) representation."""
+
+    has_z_values: bool = False
```

Comment:
I would say this is a property of the `PolyFile`, though I'm not sure if any users would actually be interested in this.

Reply:
I would argue this is a property of a specific {{PolyFile}} which indicates whether all points in this PolyFile will have a z-value or not. Because of this I opted to create a public attribute. I'm not sure if this is a good enough reason, and if you prefer it being a private variable I will change it.